athena/tests/modules/text/module_text_llm/mock/conftest.py

```
@@ -1,51 +1,67 @@
+    """
+    fake_model_config = FakeModelConfig(
+        model_name="fake_model",
+        provider="openai",
+        max_tokens=4000,
+        temperature=0.0,
+        top_p=1.0,
+        presence_penalty=0.0,
+        frequency_penalty=0.0,
     )
+
+    return DefaultApproachConfig(model=fake_model_config)
```

Ruff (0.13.3)

Undefined name `DefaultApproachConfig`

(F821)